File: Movie/Movie.Web/Modules/ServerTypes/Club/ClubForm.ts
import { StringEditor, DateEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface ClubForm {
    ClubName: StringEditor;
    Code: StringEditor;
    PingTime: DateEditor;
}

export class ClubForm extends PrefixedContext {
    static readonly formKey = 'Club.Club';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!ClubForm.init)  {
            ClubForm.init = true;

            var w0 = StringEditor;
            var w1 = DateEditor;

            initFormType(ClubForm, [
                'ClubName', w0,
                'Code', w0,
                'PingTime', w1
            ]);
        }
    }
}

File: Movie/Movie.Web/Modules/Club/Club/ClubForm.cs
using Serenity.ComponentModel;
using System;

namespace Movie.Club.Forms;

[FormScript("Club.Club")]
[BasedOnRow(typeof(ClubRow), CheckNames = true)]
public class ClubForm
{
    public string ClubName { get; set; }
    public string Code { get; set; }
    public DateTime PingTime { get; set; }
}